templates/top_config_template.yaml

```diff
@@ -0,0 +1,92 @@
+# Top Level Config Template for both the training and metric experiments
+
+# Config type, this can either edited or given as an argument in the gen_configs script
+config_type: both
+
+# Where to put the subconfigs
+config_dir: configs
+
+# Where (and if) to save results locally
+save_dir: results_pets
+local_save: False
+
+# Where to find the slurm template
+slurm_template_path: templates/
+slurm_template_name: jobscript_template.sh
+slurm_template_extension: sh  # extension required for testing purposes
```

Comment:
Could we ditch the need for `slurm_template_extension`? E.g. if the extension is needed separately somewhere later it could be extracted from the filename above.

Reply:
could also combine all three into one, but up to you

Reply:
I've removed `slurm_template_extension` and rewritten the tests to function without it.

The other two are required to be separate as `jinja` needs a directory which contains the templates to create an environment from, then a separate path to find the specific template.